src/test/java/com/facebook/genAlgo/mutator/MutatorFactoryTest.java

```diff
@@ -104,17 +104,123 @@ public void shouldReturnSingleMutatorWhenGetMutatorIsEvokedWithZeroEnumParameter
         assertThat(mutator).isInstanceOf(SingleMutator.class);
     }
 
-    @DisplayName("Should return SingleMutator with mutationChance equal 0 when getMutator() with ZERO enum as parameter is called")
+    @DisplayName("Should throw exception when getMutator() with ZERO enum as parameter is called")
     @Test
-    public void shouldReturnSingleMutatorWithZeroMutationChanceWhenGetMutatorIsEvokedWithZeroEnumParameter() {
+    public void shouldThrowExceptionWhenMutatorWithZeroMutationChanceWhenGetMutatorIsEvokedWithZeroEnumParameter() {
         // given
         MutatorFactory mutatorFactory = new MutatorFactory();
         float mutationChance = 0.3f;
 
+        // then
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> mutatorFactory.getMutator(mutationChance, MutatorEnum.ZERO));
+    }
+
+    @DisplayName("Should return instance of MultipleMutator when getMutator() with single parameter MULTIPLE enum is called")
+    @Test
+    public void shouldReturnInstanceOfMultipleMutatorWhenGetMutatorWithSingleParamIsEvoked() {
+        // given
+        MutatorFactory mutatorFactory = new MutatorFactory();
+        float mutationChance = 0.2f;
+
         // when
-        MutatorService mutator = mutatorFactory.getMutator(mutationChance, MutatorEnum.ZERO);
+        MutatorService mutator = mutatorFactory.getMutator(MutatorEnum.MULTIPLE);
+
+        // then
+        assertThat(mutator).isInstanceOf(MultipleMutator.class);
+    }
+
+    @DisplayName("Should return instance of MultipleMutator with default mutationChance when getMutator() with single parameter MULTIPLE enum is called")
+    @Test
+    public void shouldReturnInstanceOfMultipleMutatorWithDefaultMutationChanceWhenGetMutatorWithSingleParamIsEvoked() {
+        // given
+        MutatorFactory mutatorFactory = new MutatorFactory();
+
+        // when
+        MutatorService mutator = mutatorFactory.getMutator(MutatorEnum.MULTIPLE);
 
         // then
-        assertThat(((SingleMutator)mutator).getMutationChance()).isEqualTo(0);
+        assertThat(((MultipleMutator)mutator).getMutationChance()).isEqualTo(0.05f);
```

Comment:
Since we are checking default value, I think we should have public static final float DEFAULT_MUTATION_CHANCE declared somewhere.